src/isPalindrome.js

```diff
@@ -0,0 +1,11 @@
+export default function isPalidrome (str){
+  var re = /[^A-Za-z0-9]/g;
+  str = str.toLowerCase().replace(re, '');
```

Comment:
Avoid overwriting inputs - this has the potential to have inadvertent consequences outside of this function (known as side effects) if the calling context was not expecting the input to be changed.  I would recommend declaring a local variable to hold the lower cased string for processing in this context.